src/gui/sdlmain.cpp

```diff
@@ -2169,61 +2167,31 @@ void GFX_SetShader([[maybe_unused]] const std::string &source)
 #endif
 }
 
-bool GFX_MouseIsAvailable() {
-	return sdl.mouse.control_choice != NoMouse;
-}
-
 void GFX_SetMouseRawInput(const bool raw_input)
 {
-    if (!SDL_SetHintWithPriority(SDL_HINT_MOUSE_RELATIVE_MODE_WARP,
-                                 raw_input ? "0" : "1",
-                                 SDL_HINT_OVERRIDE))
-    	LOG_WARNING("DISPLAY: Mouse raw input set failed");
+	if (SDL_SetHintWithPriority(SDL_HINT_MOUSE_RELATIVE_MODE_WARP,
+	                            raw_input ? "0" : "1",
+	                            SDL_HINT_OVERRIDE) != SDL_TRUE)
+		LOG_WARNING("SDL: Mouse raw input %s failed",
+		            raw_input ? "enable" : "disable");
 }
 
-void GFX_ToggleMouseCapture()
+void GFX_SetMouseCapture(const bool capture)
 {
-	/*
-	 * Only process mouse events when we have focus.
-	 * This protects against out-of-order event issues such
-	 * as acting on clicks before the window is drawn.
-	 */
-	if (!sdl.mouse.has_focus)
-		return;
-
-	assertm(sdl.mouse.control_choice != NoMouse,
-	        "SDL: Mouse capture is invalid when NoMouse is configured [Logic Bug]");
-
-	mouse_is_captured = !mouse_is_captured; // flip state
-	if (SDL_SetRelativeMouseMode(mouse_is_captured ? SDL_TRUE : SDL_FALSE) != 0) {
+	if (SDL_SetRelativeMouseMode(capture ? SDL_TRUE : SDL_FALSE) != 0) {
```

Comment:
`capture` is an action, and on it's own doesn't convey boolean-ness.

`should_capture` -> should we capture?
`is_captured` -> is it current captured?
`captured` -> was it captured in the past?

These are all more self-documenting about "booleaness".

Suggest always naming boolean variables like this:
`is_<some state>`
`should_<some state>`
`<some state>_requested`
(etc..)

Reply:
(even growing up as a native english speaker, I only just learned that these words fall under category called "state of being", which seem to be a good fit for boolean variables :joy:)

Reply:
Changed to `request_*` (`request_capture`, etc.). I hope this is explanatory enough.

Reply:
I've lost context to what `request_capture` was: is this a static function that lets us request capturing? (if so: function name looks good).

If it's a bool, then the name can't be a verb.  Suggest:
 - `requested_capture`, or
 - `capture_was_requested`

Reply:
`requested_*` seems to be the best. Also added some comments in `mouse.h` to explain purpose of these `Notify*` functions.